app/lib/bk/compat/parsers/circleci/orbs/docker/install_dockerize.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module BK
+  module Compat
+    module CircleCISteps
+      # Handles the translation of docker installation commands
+      class DockerOrb
+        def translate_docker_install_dockerize(config)
+          install_dir = config.fetch('install-dir', '/usr/local/bin')
+          version = config.fetch('version', 'latest')
+          [
+            '# Instead of installing dockerize in a step, we recommend your agent environment to have it pre-installed',
+            "echo '~~~ Installing Dockerize'",
+            (install_dir == '/usr/local/bin' ? [] : ["mkdir -p #{install_dir}"]),
+            install_dockerize_commands(version, install_dir)
+          ].flatten.compact
+        end
+
+        private
+
+        def install_dockerize_commands(_version, install_dir)
+          ['PLATFORM=""',
+           'if [ "$(uname -s)" = "Darwin" ]; then',
+           '  PLATFORM="darwin-amd64"',
+           'else',
+           '  SYS_ARCH=$(uname -m)',
+           '  case "$SYS_ARCH" in',
```

Comment:
in the docker compose script this is done with a regex, is there any reason for it to be different?